recipes/hepconvert/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "hepconvert" %}
+{% set version = "1.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hepconvert-{{ version }}.tar.gz
+  sha256: 2c426a9b5fa36959275098bc044d93486df81efefc9777ba2affd373e8bbfc19
+
+build:
+  entry_points:
+    - hepconvert = hepconvert.__main__:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - hatchling
+    - hatch-vcs
+    - pip
+  run:
+    - python >=3.8
+    - uproot >=5.0.0
+    - click >=8.1
+    - click-default-group >=1.2
+
+test:
+  imports:
+    - hepconvert
+  commands:
+    - pip check
+    - hepconvert --help
+  requires:
+    - pip
+
+about:
+  summary: File conversion package.
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - matthewfeickert
+    - zbilodea
+    - jpivarski
```

Comment:
@zbilodea @jpivarski Can you both comment on the PR if you would be willing to be conda-forge recipe maintainers with me for `hepconvert`?

Reply:
That's fine with me!

Reply:
@zbilodea Great! The way the linter works though, it requires you to comment in the main PR thread just saying that you're willing to be a maintainer.

Reply:
I see!